### DISCUSSION TITLE
PLEASE implement --select

### DISCUSSION THREAD

**Discussion by yavnest** [linuxmint/discussions#150] [Ideas]
This is from **nautilus**'s manpage, which I'm now forced to use since **nemo** doesn't have that feature:
```
-s, --select
              Select specified URI in parent folder.
```

This allows programs (like qBittorrent, for example) to open the file manager _with a specific file SELECTED._

If I set the default file manager to **nautilus**, (since it has --select) when I right-click a specific file of a torrent and choose "Open containing folder" in the context menu, it opens the file manager in its parent directory _with the file selected!_
Now, if **nemo** is the default file manager:  my ~/Downloads directory has a lot of directories and files in it, so if the file happens to be in it, every time I'm forced to search for the file I need, since it doesn't get selected; I'm at the top of the parent directory every time. It's such a bother!
This is an essential feature for a file manager, IMO.

**Comment by mtwebster** [comment]
There's already an interface specifically designed for this sort of functionality - see https://www.freedesktop.org/wiki/Specifications/file-manager-interface/. Nemo implements this (as do many other file managers, including nautilus). This is how programs like firefox perform their 'Show in folder' activation from the download menu (and would avoid qbittorrent having to maintain a hardcoded list of file managers and their arguments (https://github.com/qbittorrent/qBittorrent/blob/master/src/gui/utils.cpp#L182-L214).

I'll see about adding `--select` at some point

**Comment by yavnest** [comment]
Oh, Firefox! It didn't occur to me that I had already witnessed this open-with-certain-file-selected behavior work successfully with nemo! :) That would have told me there must exist some mechanism for it after all.

Well, it seems like the qBittorrent people are gonna have to improve their game. Probably the person who wrote it also wasn't aware of the DBus interface you mentioned.
I'll write a request on their page with a link to this discussion.

Thanks for the excellent response!
